app/models/address.rb

```diff
@@ -0,0 +1,10 @@
+class Address < ApplicationRecord
+  belongs_to :user
+
+  validate :num_digits_in_pincode
+
+  private
+    def num_digits_in_pincode
+      pincode.Math.log10(self).to_i + 1 == 6
```

Comment:
please explain what is the need to make this so complicated

Reply:
removed this and used length validation instead